spec/lib/generators/module_component/module_components_generator_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'generators/module_component/module_component_generator'
+require 'generators/module/module_generator'
+
+RSpec.describe 'ModuleComponent', type: :generator do
+  before(:all) do
+    @original_stdout = $stdout
+    # Redirect stdout to suppress generator output
+    $stdout = File.open(File::NULL, 'w')
+  end
+
+  after(:all) do
+    $stdout = @original_stdout
+  end
```

Comment:
```suggestion
  before { allow($stdout).to receive(:write) }
```

Would something like this work?